frontend/pages/game.html

```diff
@@ -17,6 +19,55 @@
         <canvas id="gameCanvas"></canvas>
         <canvas id="holdCanvas"></canvas>
     </div>
+
+    <div class = "timer">
+        <canvas id="timer"></canvas>
+    
+
+    </div>
+    <div id="div1">
+      
+      
+        <button class="click-me" onclick="startCountUp()">Start</button>
+        
+        <div id="countup-text"></div>
+      </div>
+
+    
+<script type="text/javascript">
+var isSetTimmeoutRunning = false;
+  function startCountUp(){
+
+  if( isSetTimmeoutRunning == false ){
+    isSetTimmeoutRunning = true;
+    
+    setTimeout(function(){
+      isSetTimmeoutRunning = false;
+      alert(2 + " minutes have passed.");
+    }, 120000);
+    
+    var counter = 0;
+    
+    document.getElementById("countup-text").innerHTML = "<p>" + 0 + "</p>";
+
+    var interval = setInterval(function(){
+      counter++;
+      document.getElementById("countup-text").innerHTML = "<p>" + counter + "</p>";
+      if( counter == 100000 ){
+        document.getElementById("countup-text").innerHTML = "";
+        clearInterval(interval);
+      }
+      if (document.URL.includes("gameOver.html")){

```

Comment:
don't need this bit as the code will stop when we go to another page - but good thinking!